doc/source/redis-memory-management.rst

```diff
@@ -0,0 +1,103 @@
+Redis Memory Management (EXPERIMENTAL)
+==============================================
```

Comment:
the line of `====` should have the same length as the title (this applies in other places as well)

Reply:
Done, fascinating...